Formula/subversion.rb

```diff
@@ -235,21 +240,19 @@ def caveats
     system "#{bin}/svnadmin", "create", "test"
     system "#{bin}/svnadmin", "verify", "test"
 
-    if Hardware::CPU.intel?
-      platform = "darwin-thread-multi-2level"
-      on_linux do
-        platform = "x86_64-linux-thread-multi"
-      end
-
-      perl = "/usr/bin/perl"
-      on_linux do
-        perl = "#{Formula["perl"].opt_bin}/perl"
-      end
+    platform = "darwin-thread-multi-2level"
+    on_linux do
+      platform = "x86_64-linux-thread-multi"
+    end
 
-      perl_version = Utils.safe_popen_read(perl.to_s, "--version")[/v(\d+\.\d+(?:\.\d+)?)/, 1]
-      ENV["PERL5LIB"] = "#{lib}/perl5/site_perl/#{perl_version}/#{platform}"
-      system perl, "-e", "use SVN::Client; new SVN::Client()"
+    perl = "/usr/bin/perl"
+    on_linux do
+      perl = "#{Formula["perl"].opt_bin}/perl"
```

Comment:
```suggestion
    perl = DevelopmentTools.locate("perl")
```
This should do the same thing but less verbosely.

Reply:
There's probably an `end` here that needs removing too.

Reply:
Updated, thanks.